lib/profiler/mitmDumpProfiler.py

```diff
@@ -36,8 +38,26 @@ def start_recording(self):
                 replay_cmd_list = ["mitmdump", "--no-http2", "-S", flow_file_fp, "--no-pop", "--norefresh"]
             self.process = subprocess.Popen(replay_cmd_list)
 
+        if sys.platform == "linux2":
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy", "mode", "manual"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.http", "host", "127.0.0.1"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.http", "port", "8080"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.https", "host", "127.0.0.1"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.https", "port", "8080"])
+        else:
```

Comment:
If we don't have the way to start recording on `darwin`, we can raise `NotImplementedError`
Ref: https://docs.python.org/2/library/exceptions.html#exceptions.NotImplementedError